web/src/engine/websites/ArgosScan.ts

```diff
@@ -0,0 +1,180 @@
+import { Tags } from '../Tags';
+import icon from './ArgosScan.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchGraphQL, FetchRequest } from '../FetchProvider';
+
+type APIMangas = {
+    getProjects: {
+        projects: {
```

Comment:
Can probbly be simplified by using type `APIManga []`

Reply:
Nope; ApiManga got the infos wrapped inside ```project : {```` .  It wont fit.

```
type APIManga = {
    project: {
        name: string,
        id: number,
```

```
type APIMangas = {
    getProjects: {
        projects: {
            id: number,
            name: string
```